modules/gdscript/gdscript_parser.cpp

```diff
@@ -1530,7 +1530,10 @@ GDScriptParser::SuiteNode *GDScriptParser::parse_suite(const String &p_context,
 				VariableNode *variable = static_cast<VariableNode *>(statement);
 				const SuiteNode::Local &local = current_suite->get_local(variable->identifier->name);
 				if (local.type != SuiteNode::Local::UNDEFINED) {
-					push_error(vformat(R"(There is already a %s named "%s" declared in this scope.)", local.get_name(), variable->identifier->name));
+					Node *errLoc = alloc_node<PassNode>();
```

Comment:
Godot tends to prefer snake_case for variable names. The C++ formatting tries to follow the [GDScript Formatting Guidelines](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html?highlight=snake_case#naming-conventions) where applicable.

Reply:
ohk, will keep that in mind.